packages/medusa/src/api/routes/admin/orders/create-claim.ts

```diff
@@ -437,6 +437,10 @@ export class AdminPostOrdersOrderClaimsReq {
   @IsOptional()
   no_notification?: boolean
 
+  @IsOptional()
+  @IsString()
+  location_id?: string
```

Comment:
suggestion: should we rename to `return_location_id`? To make it clear that the location is independent of the claim fulfillment.